oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -228,13 +248,25 @@ static FullGCMode getFullGcMode() {
                             final boolean fullGCEnabled,
                             final boolean isFullGCDryRun,
                             final boolean embeddedVerification) {
+        this(nodeStore, gcSupport, fullGCEnabled, isFullGCDryRun, embeddedVerification, 0);
```

Comment:
please use `Configuration.DEFAULT_FULL_GC_MODE` instead of 0

Reply:
fixed, thank you for the comment. I used DocumentNodeStoreService.DEFAULT_FULL_GC_MODE